File: text/ringo/read/text_read.js
#! /usr/bin/ringo
// ------------------------------------------------------------------
//
//	text_read.js
//
//						May/09/2011
//
// ------------------------------------------------------------------
var text_manipulate = require ('/var/www/data_base/common/ringo_common/text_manipulate.js');
// ------------------------------------------------------------------
print	("*** 開始 ***");
var args = require('system').args;
var file_in = args[1];

var dict_aa = text_manipulate.text_read_proc (file_in);
text_manipulate.dict_display_proc (dict_aa);

print	("*** 終了 ***");
// ------------------------------------------------------------------

File: text/jruby/read/text_read.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	read/text_read.rb
#
#					Jan/21/2011
#
# ---------------------------------------------------------------------
load '/var/www/data_base/common/ruby_common/text_manipulate.rb'

# ---------------------------------------------------------------------
file_in = ARGV[0]
#
puts "*** 開始 ***"
#
dict_aa = text_read_proc(file_in)
dict_display_proc(dict_aa)
#
puts "*** 終了 ***"
# ---------------------------------------------------------------------
